src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -678,11 +685,11 @@ const DateTimePicker = ({
           <div className={`${iotPrefix}--date-time-picker__menu-scroll`}>
             {!isCustomRange ? (
               <OrderedList nested={false}>
-                {getIntervalValue() ? (
+                {renderPresetTooltipText(currentValue) || getIntervalValue() ? (
                   <ListItem
                     className={`${iotPrefix}--date-time-picker__listitem ${iotPrefix}--date-time-picker__listitem--current`}
                   >
-                    {getIntervalValue()}
+                    {renderPresetTooltipText(currentValue) || getIntervalValue()}
```

Comment:
I think these can stay as they are actually. I don't think we need the tooltipText here since this is the actual list and not the tooltip?

Reply:
That's for displaying the internal value at the top of the expanded div. We do want to show the same text as the tooltip.

Reply:
@tay1orjones Can I mark this conversation resolved?